Tests/AppcuesKitTests/Experiences/ExperienceStateMachine+AnalyticsObserverTests.swift

```diff
@@ -14,59 +14,63 @@ class ExperienceStateMachine_AnalyticsObserverTests: XCTestCase {
     var appcues: MockAppcues!
     private var updates: [TrackingUpdate] = []
     var observer: ExperienceStateMachine.AnalyticsObserver!
+    var analyticsExpectation: XCTestExpectation!
 
     override func setUpWithError() throws {
         appcues = MockAppcues()
         appcues.analyticsPublisher.onPublish = { update in
             self.updates.append(update)
+            self.analyticsExpectation?.fulfill()
         }
         observer = ExperienceStateMachine.AnalyticsObserver(container: appcues.container)
     }
 
-    override func tearDownWithError() throws {
+    override func tearDown() async throws {
```

Comment:
in various spots like this could we update to `override func tearDown()` ? not important, just curious, if we need the throwing cases

and L19 `override func setUp() {`

Reply:
It's a bit goofy how XCTest has evolved over the years. You can do any of
```swift
func setUp()
```